src/main/kotlin/com/chattriggers/ctjs/engine/IRegister.kt

```diff
@@ -724,12 +724,12 @@ interface IRegister {
         return RegularTrigger(method, TriggerType.POST_GUI_RENDER, getImplementationLoader())
     }
 
+    // TODO: is there any way to get the ParticleType from a Particle?
```

Comment:
This doesn't seem possible, so lets just commit to it and change to `TODO(breaking)`